spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageResource.java

```diff
@@ -146,4 +136,14 @@ public String getDescription() {
 	public InputStream getInputStream() throws IOException {
 		return Channels.newInputStream(resolve().reader());
 	}
+
+	@Override
+	public boolean isWritable() {
+		return true;
+	}
+
+	@Override
+	public OutputStream getOutputStream() throws IOException {
+		return new GoogleStorageOutputStream(resolve());
```

Comment:
I didn't realize that the `com.google.cloud.WriteChannel` actually implements `java.nio.channels.WritableByteChannel` and we can use the `Channels.newOutputStream`. This is actually much better! Thanks!